translations/base-de.yaml

```diff
@@ -382,7 +379,7 @@ shopUpgrades:
 buildings:
     hub:
         deliver: Liefere
-        toUnlock: "Für die Belohnung:"
+        toUnlock: "zum freischalten:"
```

Comment:
Das steht ja im Hub, ich glaube so wirkt das etwas komisch: "Zum Freischalten: Verteiler"

Reply:
In der Beta ist das zu lang gewesen
![grafik](https://user-images.githubusercontent.com/57270769/94908785-835c2800-04a2-11eb-9396-bfd66aaee74c.png)

Reply:
Oh, mir fällt grad auf, dass das gar nicht gleich ist

Reply:
was hälst du von `und schalte frei:`? Ansonsten ändere ichdas zurück

Reply:
Hört sich gut an!